Lib/idlelib/idle_test/example_stub.pyi

```diff
@@ -1,4 +1,4 @@
-" Example to test recognition of .pyi file as Python source code.
+"""Example to test recognition of .pyi file as Python source code."""
```

Comment:
```suggestion
"""An example to test the recognition of .pyi files as Python source code"""
```
LGTM

Reply:
Wulian is correct below that `"` should have been `#`.  I will resolve this and make a better fix.